src/libraries/System.Formats.Nrbf/ref/System.Formats.Nrbf.csproj

```
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppPrevious);$(NetCoreAppMinimum);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
+    <CLSCompliant>false</CLSCompliant>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="System.Formats.Nrbf.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'">
```

@adamsitnik  The error is saying that the namespace is not being found for .NET Standard 2.0. The problem is that you're conditioning that ItemGroup to only compile for NetCoreAppCurrent. The solution is to condition it to include those projects in both.

I tried this and it worked:

```suggestion
  <ItemGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'netstandard2.0'))">
```

The 3 csprojs will be able to build (note: the next errors that you will have to investigate is the ApiCompat errors, but those are unrelated).

@ViktorHofer can confirm is my suggestion is the most appropriate or if there's a better one.

---

> $([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'netstandard2.0'))

When would this condition be false? All TFMs targeted by this library should be compatible with netstandard2.0.